sonosco/datasets/audio_dataset.py

```diff
@@ -55,19 +52,24 @@ def retrieve_file(audio_path):
         return sound, sample_rate
 
     @staticmethod
-    def augment_audio(sound, tempo_range: Tuple = (0.85, 1.15), gain_range: Tuple = (-6, 8)):
-        """Changes tempo and gain of the wave."""
-        warnings.warn("Augmentation is not implemented")  # TODO: Implement
-        return sound
+    def augment_audio(sound):
```

Comment:
Should be a separate AudioEffects interface which may be called here?

Reply:
will add this later